analyzers/src/SonarAnalyzer.CSharp/SonarAnalyzer.CSharp.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <LangVersion>8.0</LangVersion>
+    <LangVersion>9.0</LangVersion>
```

Comment:
I will change the other projects lang version (and move it to Directory.Build.props) in a future PR